docker-compose.yml

```diff
@@ -345,21 +399,30 @@ services:
     networks:
       n4htoleafbn2:
     command: |
-      sh -x -c 'sleep 30 && \
```

Comment:
why sleep is removed ?

Reply:
I see that BGP sessions are established quickly, we dont need any delay.